router/core/router.go

```diff
@@ -2198,3 +2199,19 @@ func or[T any](maybe *T, or T) T {
 	}
 	return or
 }
+
+func isNotDefaultCloudExporter(metricConfig *rmetric.Config) bool {
```

Comment:
I think we can put this into the metric package. I'd avoid negation. How about `HasCloudExporterConfigured`